### PR TITLE
Decommission new interactives functionality

### PR DESCRIPTION
**Pull request by jondewijones**
### What

- Decommission new interactives functionality
  - remove config
  - remove code/components for interactive functionality 
  - remove axios (only used by interactives)
- Upgrade golang.org/x/net to 0.17.0

### Who can review

Anyone

### CODE REVIEW

File: src/audit-allowlist.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
             "reason": "indirect dependency of a few babel packages. should be looked at as a seperate ticket. see: https://trello.com/c/alXeUZpI/1918-fix-dependency-issues-with-babel-traverse"
         },
         {
-            "id": "CVE-2023-45857",
-            "reason": "axios is only used by interactives and will be removed soon. see: https://trello.com/c/zub5ECee/1916-remove-interactive-api-code-config-from-florence-and-dp-configs"
+            "id" : "CVE-2022-25883",
+            "reason": "Semver"
```
Comment (lindenmckenzie) [inline src/audit-allowlist.json:25]: Can you provide a bit more of an explanation of why this has been allowed?

**Review by lindenmckenzie — COMMENTED** [review]
Minor comment.

File: src/audit-allowlist.json (hunk 1 above)
Comment (jondewijones) [reply src/audit-allowlist.json:25]: Wish i could remember why. It was added months ago https://github.com/ONSdigital/florence/pull/989/files

I'll have a look at the details and try add something more helpful

**Review by lindenmckenzie — DISMISSED** [review]

**Review by gedge — APPROVED** [review]
re-approving with develop update

**Pull request merged**